etools-refresh-behavior.html

```diff
@@ -0,0 +1,73 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../etools-dexiejs/etools-dexiejs.html">
```

Comment:
There is a new `dexie.html` file I created not log ago. We can use it to import DexieJS library. No need to use `etools-dexiejs.html` element anymore.